distribution/lambda/Makefile

```diff
@@ -0,0 +1,68 @@
+.SILENT:
+.ONESHELL:
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+
+check-env:
+ifndef CDK_ACCOUNT
+	$(error CDK_ACCOUNT is undefined)
+endif
+ifndef CDK_REGION
+	$(error CDK_REGION is undefined)
+endif
+
+build:
+	cargo lambda build --manifest-path=../../quickwit/Cargo.toml -p quickwit-lambda --release
+
+bootstrap: build check-env
+	cdk bootstrap aws://$$CDK_ACCOUNT/$$CDK_REGION
+
+generate-base-stack:
+	cdk synth -a cdk/app.py BaseQuickwitStack
+
+deploy-hdfs: build check-env
+	cdk deploy -a cdk/app.py HdfsStack
+	
+invoke-hdfs-indexer: check-env
+	python -c 'from cdk import cli; cli.upload_hdfs_src_file()'
+	python -c 'from cdk import cli; cli.invoke_hdfs_indexer()'
+
+invoke-hdfs-searcher-severity: check-env
+	python -c 'from cdk import cli; cli.invoke_hdfs_searcher("""{"query": "severity_text:ERROR"}""")'
+
+invoke-hdfs-searcher-histogram: check-env
+	python -c 'from cdk import cli; cli.invoke_hdfs_searcher("""{ "query": "*", "max_hits": 0, "aggs": { "events": { "date_histogram": { "field": "timestamp", "fixed_interval": "1h" }, "aggs": { "log_level": { "terms": { "size": 10, "field": "severity_text", "order": { "_count": "desc" } } } } } } }""")'
```

Comment:
A fixed interval of 1h is not very relevant on this dataset. I would set it to 1d.